hs/server.hs

```diff
@@ -44,90 +19,9 @@ main = do
     let staticDir = encodeString $ parent $ decodeString cwd
     contents <- readFile "../README.md"
     simpleHTTP nullConf $
-      msum [ dir grid $ gridResponse,
-             dir graph $ graphResponse,
-             dir about $ aboutResponse contents,
-             dir static $ serveDirectory EnableBrowsing [] staticDir,
-             dir course $ path (\s -> liftIO $ queryCourse s)
+      msum [ dir "grid" $ gridResponse,
```

Comment:
Done. I tried this before, but for some reason it didn't work. Worked this time. Hmm.